dask_cuda/proxy_object.py

```diff
@@ -237,28 +214,20 @@ def __init__(
         subclass: Optional[bytes],
         serializer: Optional[str],
         explicit_proxy: bool,
+        manager: Union["ProxyManager", ProxyManagerDummy] = ProxyManagerDummy(),
```

Comment:
Yes, `ProxyManager` is imported only when type checking (to avoid cyclic imports) so it is not defined at runtime.
```python
if TYPE_CHECKING:
    from .proxify_host_file import ProxyManager
```